examples/reward_model.py

```diff
@@ -0,0 +1,108 @@
+from transformers import GPT2LMHeadModel, GPT2Tokenizer, GPT2Model, PreTrainedModel, AutoModelForCausalLM, GPT2PreTrainedModel, GPT2Model
```

Comment:
We will probably want to keep the reward model code somewhere separate. Likely in the magiCarp repo.